php/blocks/class-loader.php

```diff
@@ -132,8 +132,7 @@ protected function get_callback( $method_name ) {
 	 * Launch the blocks inside Gutenberg.
 	 */
 	protected function editor_assets() {
-		$asset_config_file = $this->plugin->get_path( 'js/editor.blocks.asset.php' );
-		$asset_config      = require $asset_config_file;
+		$asset_config = require $this->plugin->get_path( 'js/editor.blocks.asset.php' );
```

Comment:
This isn't related to this PR, it's just something I noticed could be more concise.